migrations/245_pl_course__question_params__add.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE pl_courses ADD COLUMN question_params jsonb DEFAULT '{}'::jsonb;
```

Comment:
IMO we should batch all these up into a single file, unless there's a particular reason they're separate now.